docs/sources/k6/next/javascript-api/k6-browser/metricmessage.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'MetricMessage'
+slug: 'metricmessage'
```

Comment:
```suggestion
```

If the URL is going to be the same name as the filename, you don't need the `slug` property here. :)

Reply:
Sure, removed in 3baf38691cea0e84ba92bad652d60e47bed7117b